docs/decisions/0004-downstream-effect-events.rst

```diff
@@ -0,0 +1,103 @@
+Implementation of Event Driven Architecture for Exam Downstream Effects
+=======================================================================
+
+Status
+------
+
+Draft (circa Sept 2023)
+
+Context
+-------
+For Milestone 6 of the exams IDA project, we are porting any downstream effects of exam submission and review
```

Comment:
You are referencing the plans and name of the project, but you did not reference a project overview/summary. Imagine the confusion for others outside of Cosmonauts team.
Either don't reference the project, or provide a link to the project overview document